fvalela1/mod-calculator
# Editable text views

Resolves [#29](https://github.com/fvalela1/mod-calculator/issues/29)
## How to test changes
- enter bunch of numbers
- tap on either formula view or result view
- It should not be editable
